airflow/utils/log/file_task_handler.py

```diff
@@ -207,7 +207,7 @@ def read(self, task_instance, try_number=None, metadata=None):
             logs = [
                 [('default_host', f'Error fetching the logs. Try number {try_number} is invalid.')],
             ]
-            return logs
+            return logs, [{'end_of_log': True}]
```

Comment:
This is interesting because such error should be spotted by `pylint` as inconsistent return statement. I think this is yet another time when I have a feeling that our pylint may not work perfectly well. CC @kaxil @ashb @potiuk